spec/factories/drug_stocks.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :drug_stock do
+    id { SecureRandom.uuid }
+    association :facility, strategy: :create
+    association :user, strategy: :create
+    association :protocol_drug, strategy: :create
+    recorded_at { Time.current }
+    in_stock { 500 }
+    received { 500 }
```

Comment:
Let's make these numbers different for better entropy in our tests. Might as well make the `received` number significantly less than `in_stock` as well while we're at it, for conceptual accuracy (~100)